neutrino.go

```diff
@@ -498,9 +511,7 @@ func (sp *ServerPeer) OnRead(_ *peer.Peer, bytesRead int, msg wire.Message,
 	sp.mtxSubscribers.RLock()
 	defer sp.mtxSubscribers.RUnlock()
 	for subscription := range sp.recvSubscribers {
-		subscription.wg.Add(1)
```

Comment:
Why remove the wait group in this case?

Reply:
It was causing a data race due to the incorrect way it was being used by the subscription. Those goroutines don't need to shut down cleanly after the quit channel is closed, so I just removed it.